sketch.js

```diff
@@ -0,0 +1,47 @@
+var colony;
```

Comment:
I don't think it is a good idea to create these general variables outside of the setup functions. I think they could be set once and then passed into the draw function where you need them.